include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_comp_v3.hpp

```diff
@@ -62,9 +64,13 @@ struct GemmPipelineAgBgCrCompV3 : public BaseGemmPipelineAgBgCrCompV3<Problem>
     static constexpr index_t NPerBlock = BlockGemmShape::kN;
     static constexpr index_t KPerBlock = BlockGemmShape::kK;
 
-    static constexpr index_t VectorSizeA = Policy::template GetVectorSizeA<Problem>();
-    static constexpr index_t VectorSizeB = Policy::template GetVectorSizeB<Problem>();
-    static constexpr index_t VectorSizeC = Policy::template GetVectorSizeC<Problem>();
+    static constexpr index_t GetVectorSizeA() { return Policy::template GetVectorSizeA<Problem>(); }
+    static constexpr index_t GetVectorSizeB() { return Policy::template GetVectorSizeB<Problem>(); }
+    template <bool EpilogeWithTranspose = false>
+    static constexpr index_t GetVectorSizeC()
+    {
+        return Policy::template GetVectorSizeC<Problem, EpilogeWithTranspose>();
+    }
```

Comment:
I don't like this. I think we shouldn't make here such "dependencies". You have all required information in kernel class and there you should make appropriate decisions.

Reply:
So how can we deduce vector size without knowledge about layout after epliogue? We can discuss it